### ISSUE TITLE
The display does not work after the firmware update

### ISSUE THREAD

**Issue by ZekMan** [UnifiedEngineering/T-962-improvements#115]
After the update, the display stopped working. The backlight of the display blinks after the stove is turned on. After that, nothing is displayed on the displays. 

I was also stupid enough not to make a dump of the original firmware. =\

Can you share the original firmware or help with the display?

**Comment by balcer** [comment]
I have old MCU with original Chinese firmware. It will be difficult but I can make dump of the hex. 
Did you tried to flash the uC with unpluged display. This was the problem in my case. I think the flashing interface is also connected to display.

**Comment by GitLang** [comment]
Is it possible you have left the connection that puts it into programming mode, maybe by leaving a programming lead attached?. That is exactly the effect you would see.

**Comment by ZekMan** [comment]
@GitLang 
I tried to put the controller into the firmware mode and reset it again (Pull n_RESET low briefly). If you talk about this.
`Ground n_ISP
Ground and release n_RESET
Release n_ISP
Activate LPC2000 Flash Utility on the correct COM port
Select Read Device ID and hit OK/Enter to dismiss the "please reset your board now" dialog.
Dump flash or Upload new hex file
Change LPC2000 COM-port to a different port (doesn't matter which one), as otherwise the program will continue to hog the COM port and you can't hook up a terminal program to it.
Pull n_RESET low briefly again to start the software again.`

**Comment by GadgetGroup** [comment]
Hi, I just bricked my T-962 it seems.
I have uploaded the firmware 0.51 on a LPC2144.
The firmware is loaded without any problems, but the display is all white, with no text.
Heater and fan comes on when power is turn on.

If any one have a original hex it would be lovely :o)

**Comment by mattmccrink** [comment]
Any updates on this? I just bricked my T-962 as well. Serial data seems to be streaming correctly, but LCD is blank...

**Comment by JohnWestlake** [comment]
My oven (the larger T962C) has a LPC2134 - are you sure your oven uses a LPC2144?

**Comment by GadgetGroup** [comment]
Yes, it is a LPC2144,  so if there is no response for a new hex file, I believe I need to begin to see if someone have made an upgrade kit or maybe buy a http://www.zallus.com/product/zallus-learning-controller-kit/ if the good guy is still in business.

**Comment by GadgetGroup** [comment]
Or a Controleo3

**Comment by JohnWestlake** [comment]
I just ordered a smaller T962 oven (the smallest unit) - here is the HEX dump (I hope I copied all the required memory blocks).

https://www.dropbox.com/s/b6kvk0z91dchwax/T962%20Original%20MiniOven.hex?dl=0

**Comment by GadgetGroup** [comment]
Perfect thanks, I will try to flash the oven.
I just purchased the Controleo3 on eBay, just the basic kit.
I need to have the oven up and running as soon as possible.

**Comment by romansavrulin** [comment]
@JohnWestlake , Please tell what tool did you use to dump the firmware?

**Comment by xnk** [comment]
My old T962 LPC was not read protected in any way, a simple read from Flash magic was all that was needed. But it sounds really strange that the firmware wouldn't even boot properly. Does the board still look the same as in the pictures here? Any output on the serial link (the one used for programming)?

**Comment by xnk** [comment]
Looking through the tested configurations, there are reports of successful LPC2144 installations so is really _should_ behave properly.. If there's any output on the serial port when booting that would be very interesting to see.

**Comment by eschercode** [comment]
I'm getting this bug too (with v0.52 firmware), but out of 15 boots or so, 2 have successfully displayed the Unified Engineering logo & UI works successfully. The rest of the boots the LCD is just backlight, however pressing buttons gets a reaction out of the oven - eg I can start and stop a bake (though I don't know which profile I selected, obviously). Hence, this is probably an issue with driving the display rather than bricking the MCU. Or possible duplicate of [issue 186](https://github.com/UnifiedEngineering/T-962-improvements/issues/186).
